src/main/java/org/traccar/geocoder/GeoPortailGeocoder.java

```diff
@@ -0,0 +1,53 @@
+package org.traccar.geocoder;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+/**
+ * Created by goebel on 4/17/19.
```

Comment:
Remove this. If you want, you can add license attribution like all other files.

Reply:
Done